testsuite/test_cim_obj.py

```diff
@@ -358,546 +350,627 @@ def runtest_dict(self, obj, exp_dict):
         ),
-        (
-            "Verify keybinding with timedelta value",
-            dict(
+        None, None, True
+    ),
+    (
+        "Verify keybinding with timedelta value",
+        dict(
+            init_args=[],
+            init_kwargs=dict(
                 classname='CIM_Foo',
-                keybindings=dict(K1=timedelta1_obj)),
-            dict(
+                keybindings=dict(K1=TIMEDELTA1_OBJ)),
+            exp_attrs=dict(
                 classname=u'CIM_Foo',
-                keybindings=NocaseDict(K1=timedelta1_obj)),
-            None, True
+                keybindings=NocaseDict(K1=TIMEDELTA1_OBJ)),
         ),
-        (
-            "Verify keybinding with CIMProperty (of arbitrary type and value)",
-            # Note: The full range of possible input values and types for
-            # CIMProperty objects is tested in CIMProperty testcases.
-            dict(
+        None, None, True
+    ),
+    (
+        "Verify keybinding with CIMProperty (of arbitrary type and value)",
+        # Note: The full range of possible input values and types for
+        # CIMProperty objects is tested in CIMProperty testcases.
+        dict(
+            init_args=[],
+            init_kwargs=dict(
                 classname='CIM_Foo',
                 keybindings=dict(K1=CIMProperty('K1', value='Ham'))),
-            dict(
+            exp_attrs=dict(
                 classname=u'CIM_Foo',
                 keybindings=NocaseDict([('K1', u'Ham')])),
-            None, CHECK_0_12_0
         ),
-        (
-            "Verify two keybindings",
-            dict(
+        None, None, CHECK_0_12_0
+    ),
+    (
+        "Verify two keybindings",
+        dict(
+            init_args=[],
+            init_kwargs=dict(
                 classname='CIM_Foo',
                 keybindings=OrderedDict([('K1', u'Ham'), ('K2', Uint8(42))])),
-            dict(
+            exp_attrs=dict(
                 classname=u'CIM_Foo',
                 keybindings=NocaseDict([('K1', u'Ham'), ('K2', Uint8(42))])),
-            None, True
         ),
-        (
-            "Verify case insensitivity of keybinding names",
-            dict(
-                classname='CIM_Foo',
-                keybindings=dict(Key1='Ham')),
-            dict(
+        None, None, True
+    ),
+    (
+        "Verify case insensitivity of keybinding names",
+        dict(
+            init_args=[],
+            init_kwargs=dict(
+                classname='CIM_Foo',
+                keybindings=dict(Key1='Ham')),
+            exp_attrs=dict(
                 classname=u'CIM_Foo',
                 keybindings=NocaseDict([('kEY1', u'Ham')])),
-            None, True
         ),
+        None, None, True
+    ),
 
-        # Namespace tests
-        (
-            "Verify that binary namespace is converted to unicode",
-            dict(
+    # Namespace tests
+    (
+        "Verify that binary namespace is converted to unicode",
```

Comment:
Isn't that byte namespace and not binary namespace?

Reply:
done - changed binary to bytes in all occurrences in this file, except when it was used for numbers in binary (= base 2) representations.